install_config/topics/configuring_for_azure_registry.adoc

```diff
@@ -0,0 +1,289 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-registry_{context}']
+= Configuring the {product-title} registry for Microsoft Azure
+
+Microsoft Azure provides object cloud storage that {product-title}
+can use to store container images using the {product-title} container registry.
+
+For more information, see link:https://azure.microsoft.com/en-us/services/storage/blobs/[Cloud Storage in the Azure documentation].
+
+.Prerequisites
+
+You must create a storage account to host the registry images before
+installation. The following command creates a storage account which is used
+during installation for image storage:
+
+You can use Microsoft Azure blob storage for storing container images. The
+{product-title} registry uses blob storage to allow for the registry to grow
+dynamically in size without the need for intervention from an administrator. 
+
+. Create an Azure storage account:
++
+[source,yaml]
+----
+az storage account create 
+--name <account_name> \
+--resource-group <resource_group> \
+--location <location> \
+--sku Standard_LRS
+----
++
+This creates an account key. To view the account key:
++
+[source,yaml]
+----
+az storage account keys list \
+    --account-name <account-name> \
+    --resource-group <resource-group> \
+    --output table
+
+KeyName    Permissions    Value
+key1       Full           <account-key>
+key2       Full           <extra-account-key>
+----
+
+Only one account key value is required for the configuration of the
+{product-title} registry.
+
+
```

Comment:
@bfallonf Do we need a statement here that user can configure the registry using Ansible or manually? Reading in sequence, user wouldn't know there are two ways to do this.